src/data/ACTIONS/SAM.js

```diff
@@ -287,7 +277,37 @@ export default {
 		id: 16483,
 		name: 'Tsubame Gaeshi',
 		icon: 'https://xivapi.com/i/003000/003180.png',
+		onGcd: true,
+		cooldown: 60,
+	},
+
+	KAESHI_HIGANBANA: {
+		id: 16484,
+		name: 'Kaeshi: Higanbana',
+		icon: 'https://xivapi.com/i/003000/003181.png',
+		onGcd: true,
+		cooldown: 60,
+		cooldownGroup: SAM_COOLDOWN_GROUP.TSUBAME,
```

Comment:
Does this need gcdRecast: 2.5 to avoid funky GCD uptime calculations? (Same with the other two below)

Reply:
not sure why that was reverted. but I've unreverted it